docs/probes.md

```diff
@@ -287,7 +287,7 @@ If the project does not have a badge, the probe returns one OutcomeFalse.
 
 **Lifecycle**: stable
 
-**Description**: Check that the project has an permissive license.
```

Comment:
this file is actually generated via `make generate-docs`, so can you update the corresponding `probes/<probename>/def.yml` files, run `make generate-docs` and commit the changes?